tests/application_development/code_relocation/linker_riscv_qemu_sram2.ld

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2022 The Chromium OS Authors
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @file
+ * @brief Linker command/script file
+ *
+ * Linker script for the Cortex-M platforms.
```

Comment:
Cortex-M?

Reply:
Hah, oops. I've added a commit to #49438 that fixes this.